_commands/tools/diff.md

```diff
@@ -37,5 +37,12 @@ $
 This outputs both files in 2 columns with a '|' marking lines with differences, similar to the standard output of 'sdiff'.
```

Comment:
The pipe disappears D: It'll need to be escaped with a `\`, so the text in the markdown file should look something more like `... with a '\|' marking ...`

Reply:
all your comments were resolved. Thanks for the feedback!